src/Components/Router/Router.js

```diff
@@ -0,0 +1,26 @@
+import { Routes, Route} from "react-router-dom";
+import Home from "../Home/Home";
+import Products from "../Products/Products";
+import Cart from "../Cart/Cart";
+import Wishlist from "../Wishlist/Wishlist";
+import Login from "../Form/Login/Login";
+import PageNotFound from "../PageNotFound/PageNotFound";
+import { RequireAuth } from "../Form/Authentication/RequireAuth";
+import Signup from "../Form/Signup";
```

Comment:
Suggestion: Import them from a separate index file here to make the code concise.